roles/fio-distributed/templates/configmap.yml.j2

```diff
@@ -5,9 +5,9 @@ metadata:
   name: fio-test-{{ trunc_uuid }}
   namespace: '{{ operator_namespace }}'
 data:
-{% for job in fiod.jobs %}
-{% for bs in fiod.bs %}
 {% for numjobs in fiod.numjobs %}
+{% for bs in fiod.bs %}
```

Comment:
this part of change is ideally not needed, because the order in which the files are created is not gonna affect the way we loop to trigger fio.